### PR TITLE
chore: change meilisearch-python-async to meilisearch-python-sdk

### PR DESCRIPTION
**Pull request by 7ez**
image
looking at the docs it should work, although untested

### CODE REVIEW

File: rgdps/api/__init__.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 from fastapi.responses import JSONResponse
 from fastapi.responses import Response
 from fastapi_limiter import FastAPILimiter
-from meilisearch_python_async import Client as MeiliClient
+from meilisearch_python_sdk import Client as MeiliClient
```
Comment (RealistikDash) [inline rgdps/api/__init__.py]: This should be `AsyncClient` afaik (alongside all other instances of this import).

**Review by RealistikDash — CHANGES_REQUESTED** [review]
It doesn't work.
```
realistikgdps-realistikgdps-1  |   File "/app/rgdps/api/__init__.py", line 142, in startup
realistikgdps-realistikgdps-1  |     await app.state.meili.health()
realistikgdps-realistikgdps-1  | TypeError: object Health can't be used in 'await' expression
```

File: rgdps/api/__init__.py (hunk 1 above)
Comment (7ez) [reply rgdps/api/__init__.py]: your fault for not using asyncclient

Comment (RealistikDash) [reply rgdps/api/__init__.py]: the client in `meilisearch_python_async ` was async

**Review by RealistikDash — APPROVED** [review]
Looks good

**Pull request merged**